docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc

```diff
@@ -2,26 +2,25 @@
 [role="xpack"]
 === Run {agent} on Kubernetes
 
-You can use {agent} https://www.docker.elastic.co/r/beats/elastic-agent[Docker images] on Kubernetes to
+Use {agent} https://www.docker.elastic.co/r/beats/elastic-agent[Docker images] on Kubernetes to
 retrieve cluster metrics.
 
 ifeval::["{release-state}"=="unreleased"]
 
-However, version {version} of {agent} has not yet been
-released, so no Docker image is currently available for this version.
+A Docker image is not currently available for this version, as version {version} of {agent} has not yet been released.
 
 endif::[]
 
 
 [float]
 ==== Kubernetes deploy manifests
 
-You deploy {agent} in two different ways at the same time:
+Deploy {agent} in two different ways at the same time:
 
 * As a https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/[DaemonSet]
 to ensure that there's a running instance on each node of the cluster. These
-instances are used to retrieve most metrics from the host, such as system
-metrics, Docker stats, and metrics from all the services running on top of
+instances are used to retrieve metrics from the host, such as system
+metrics, Docker stats, and metrics from all the services running on
```

Comment:
This changes the meaning too much. Not all metrics are collected from the hosts, and it is appropriate to say running on top of Kubernetes vs running on Kubernetes

```suggestion
instances are used to retrieve most metrics from the host, such as system
metrics, Docker stats, and metrics from all the services running on top of
```